Resfeber/Helpers/UITextField+Extension.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  UITextField+Extension.swift
+//  Resfeber
+//
+//  Created by Joshua Rutkowski on 12/5/20.
+//  Copyright © 2020 Spencer Curtis. All rights reserved.
+//
+
+import UIKit
+extension UITextField {
+    
+    func setInputViewDatePicker(target: Any, selector: Selector) {
+        // Create a UIDatePicker object and assign to inputView
+        let screenWidth = UIScreen.main.bounds.width
+        let datePicker = UIDatePicker(frame: CGRect(x: 0, y: 0, width: screenWidth, height: 216))//1
+        datePicker.datePickerMode = .date
+        // iOS 14 and above
+        if #available(iOS 14, *) {
```

Comment:
Because this is a new app, you could just change your deployment target to >= iOS 14.0 and that way you won't need to add these version flags anymore